File: Palindromicarray.java
public class Main {

    public static boolean isPalindrome(int[] arr) {

        int start = 0;
        int end = arr.length - number;

        while (start < end) {
            if (arr[start] != arr[end]) {
                return false;
            }
            start++;
            end--;
        }

        return true;
    }

    public static void main(String[] args) {
        int[] input = {4, 0, 8, 0, 45};

        boolean result = isPalindrome(input);

        System.out.println(result);
    }
}

File: palindromicarray.py
def is_palindrome(arr):
    start = 0
    end = len(arr) - 1

    while start < end:
        if arr[start] != arr[end]:
            return False
        start += 1
        end -= 1

    return True



input_array = [4, 0, 8, 0, 45]

result = is_palindrome(input_array)

print(result)
